### PR TITLE
Feat: Finalise cfs case note form

### PR DESCRIPTION
**Pull request by JohnFarrellDev**
**What**  
Finalise some changes for CFS case note form.

Add title field.
Move positions of tags.
Make repeater group minimized by default.

**Why**  

Based on feedback from practitioner group.

**Anything else?**

- Have you added any new third-party libraries? No
- Are any new environment variables or configuration values needed? No
- Anything else in this PR that isn't covered by the what/why? No

### CODE REVIEW

File: components/AddForm/AddForm.spec.tsx [hunk 1]
```diff
@@ -39,7 +39,7 @@ const flexibleForms: [string, boolean, boolean][] = [
   ['FACE overview assessment', false, false],
   ['Safeguarding Adult Concern Form', false, false],
   ['Safeguarding adult manager decision on concern', false, false],
-  ['Child Case Note', false, false],
+  ['Case note', false, false],
```
Comment (jhackett1) [inline components/AddForm/AddForm.spec.tsx:42]: we're replacing this component with `<AddFormDialog/>` shortly. should adjust this if that gets merged first

**Comment by JohnFarrellDev** [comment]
Add virtual visit for kind of note

Add conditional field for if child/children seen, if child/children seen alone

File: components/FlexibleForms/RepeaterGroupField.tsx [hunk 2]
```diff
@@ -35,6 +38,8 @@ const RepeaterGroupField = ({
     touched: FormikTouched<FormikValues>;
   } = useFormikContext();
 
+  const [hidden, setHidden] = useState(hiddenOnStart ?? false);
```
Comment (jhackett1) [inline components/FlexibleForms/RepeaterGroupField.tsx:41]: this seems to do the trick! though i wonder if we could do it in a stateless way by modifying line 82 on the `lib/utils/generateInitialValues.ts` file - the `getInitialValues` function. we could do something like:

```
if(field.initiallyBlank){
initialValues[field.id] = []
} else {
initialValues[field.id] = generateInitialValues(field.subfields || [], person),
}

**Review by jhackett1 — APPROVED** [review]
looks fine - left a suggestion on a possible cleaner implementation, and let's add "virtual visit" as a visit subtype

**Pull request merged**